annex/annexA.md

```diff
@@ -0,0 +1,19 @@
+ANNEX
+-----
+
+This annex is the follow-up of examples related to chapter 1 in this tutorial. 
+
+The example7.cpp is about: 
+- Listener to print the content of the event and the thread id
+- Publisher and subscriber for other types of data: int, my own struct such as Position(float _x, float _y)
+
+
+The example8.cpp is about: 
```

Comment:
Explain what is the purpose of example 8, which original example it is an extension of. Reading the code shows what it does, so this part should explain why you have added this example.

Reply:
Explanation added